src/languages/constants.ts

```diff
@@ -24,4 +25,36 @@ export const supportedLanguageIds = [
   "xml",
 ] as const;
 
+/**
+ * All language IDs that we might reference but don't have full support.
+ * (e.g., for token customization).
+ */
+export const allLanguageIds = [
```

Comment:
I tried to make this the result of adding just ["shellscript"] to `supportedLanguageIds` but the compiler didn't like it because it's constant.

Reply:
e.g. 

```typescript
/**
 * Other language IDs that we might reference (e.g., for token customization)
 * but don't have full tree-sitter support for yet.
 */
export const otherLanguageIds = [
  "shellscript",
] as const;

export const allLanguageIds = supportedLanguageIds.concat(otherLanguageIds);
```

Reply:
What about a spread?
```
export const allLanguageIds = [...supportedLanguageIds, ...otherLanguageIds];
```

Reply:
ah thanks, that's much cleaner